src/CMakeLists.txt

```diff
@@ -27,40 +49,105 @@ if(WIN32)
 
     set(RADARE2_INCLUDE_DIRS "${IAITO_WIN32_DIR}/radare2/include/libr" "${IAITO_WIN32_DIR}/include")
 endif()
+#
+# Win32 setting ends
 
-
-find_package(Radare2 REQUIRED)
-include_directories(${RADARE2_INCLUDE_DIRS})
-link_directories(${RADARE2_LIBRARY_DIRS})
```

Comment:
find_package(Radare2 1.4.0 REQUIRED)
include_directories(${RADARE2_INCLUDE_DIRS})  still are part of file please recheck. Only link_directories(${RADARE2_LIBRARY_DIRS}) was removed because it is anyways linked afterwards.